src/NewRelicExceptionHandler.php

```diff
@@ -35,36 +33,33 @@ public function __construct($ignoredExceptions = false)
         }
     }
 
-
     /**
      * @inheritdoc
      */
     public function report(Exception $e)
     {
-        if (!in_array(get_class($e), $this->ignoredExceptions)) {
-            $this->logException($e);
+        foreach ($this->ignoredExceptions as $ignored) {
+            if ($e instanceof $ignored) {
+                return false;
```

Comment:
Should this be return void instead? I don't remember the parent method signature...

Reply:
You're right, it's meant to return void, i've updated the PR.